Content.Server/Bed/Cryostorage/CryostorageSystem.cs

```diff
@@ -195,6 +196,16 @@ public void HandleEnterCryostorage(Entity<CryostorageContainedComponent> ent, Ne
             return;
         }
 
+        var cryostorageEnt = new Entity<CryostorageComponent>(cryostorageUid.Value, cryostorageComponent);
+        var enumerator = _inventory.GetSlotEnumerator(ent.Owner);
+
+        while (enumerator.NextItem(out var itemUid))
+            RaiseEventAllItems(itemUid, cryostorageEnt);
+
+        foreach (var hand in _hands.EnumerateHands(ent.Owner))
+            if (hand.HeldEntity != null)
+                RaiseEventAllItems(hand.HeldEntity.Value, cryostorageEnt);
+
```

Comment:
Just recursively iterate children rather than adding several codepaths for hands or whatever else.